tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -12,6 +12,18 @@ use tracing_core::{
 
 /// A [`Layer`] that logs formatted representations of `tracing` events.
 ///
+/// ## Examples
+///
```

Comment:
It would be nice if there was a line or two here explaining what this example demonstrates...